v1.0/deploy-a-test-cluster.md

```diff
@@ -0,0 +1,266 @@
+---
+title: Deploy a Test Cluster
+summary: Use CockroachDB's CloudFormation template to deploy a Kubernetes-orchestrated test cluster on AWS.
+toc: false
+---
+
+This page shows you the easiest way to test an insecure, multi-node CockroachDB cluster, using CockroachDB's [AWS CloudFormation](https://aws.amazon.com/cloudformation/) template to simplify setup and [Kubernetes](https://kubernetes.io/) to automate deployment, scaling, maintenance, and load balancing of client workloads.
+
+{{site.data.alerts.callout_success}}This tutorial features an older version of CockroachDB. To test the latest stable release of CockroachDB, use the <a href="../v1.1/deploy-a-test-cluster.html">v1.1 version</a> of this page. To evaluate pre-release functionality from <a href="https://github.com/cockroachdb/cockroach/wiki/Roadmap">our roadmap</a>, use the <a href="../v2.0/deploy-a-test-cluster.html">v2.0 version</a> of this page.{{site.data.alerts.end}}
+
+<div id="toc"></div>
+
+## Before You Begin
+
+Before getting started, it's important to review some limitations and requirements.
+
+### Limitations
+
+{{site.data.alerts.callout_danger}}The <a href="https://github.com/cockroachdb/cockroachdb-cloudformation">CockroachDB AWS CloudFormation template</a> is designed for testing, not for production use.{{site.data.alerts.end}}
+
+- You can scale the cluster to a maximum of 15 nodes.
+
+- While the AWS region for your deployment is configurable, the cluster runs in a single AWS availability zone within that region. It will easily survive and recover from node failures as long as you deploy at least 3 nodes, but it will not survive an availability zone outage.
+    - For production resiliency, the recommendation would be to span 3 or more availability zones in a single region or 3 or more regions.
+
+- The cluster is completely insecure, which comes with risks:
+    - There is no network encryption or authentication, and thus no confidentiality.
+    - The cluster is open to any client by default, although you have the option to restrict client access to a specific range of IP addresses.
+    - Any user, even `root`, can log in without providing a password.
+    - Any user, connecting as `root`, can read or write any data in your cluster.
+
+### Requirements
+
+- You must have an [AWS account](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-sign-up-for-aws.html).
+- You must have [SSH access](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html) in the AWS region where the cluster is deployed.   
+
+## Step 1. Start CockroachDB
+
+1. [Launch the CockroachDB CloudFormation template](http://amzn.to/2C5IFhv).
```

Comment:
Can we use this link instead? This lets us track clicks: http://amzn.to/2CZjJLZ

Reply:
Done.